geomstats/connection.py

```diff
@@ -27,12 +27,71 @@ def connection(self, tangent_vector_a, tangent_vector_b, base_point):
         raise NotImplementedError(
                 'connection is not implemented.')
 
-    def parallel_transport(self, tangent_vector_a, tangent_vector_b):
+    def exp(self, tangent_vector_a, base_point):
         """
-        Parallel transport associated with the connection.
+        Connection applied to tangent_vector_b in the direction of
+        tangent_vector_a, both tangent at base_point.
+        """
+        raise NotImplementedError(
+                'The affine connection exponential is not implemented.')
+
+    def log(self, point, base_point):
+        """
+        Connection applied to tangent_vector_b in the direction of
+        tangent_vector_a, both tangent at base_point.
         """
         raise NotImplementedError(
-                'Parallel transport is not implemented.')
+                'The affine connection logarithm is not implemented.')
+
+    def pole_ladder_transport(
+            self, tangent_vector_a, tangent_vector_b, base_point):
+        """
+        One step of pole ladder (parallel transport associated with the
+        symmetric part of the connection using transvections).
+        """
+        mid_point = self.exp(base_point=base_point,
+                             tangent_vector=1. / 2. * tangent_vector_b)
+        end_point = self.exp(base_point=mid_point,
+                             tangent_vector=- self.log(mid_point, base_point))
```

Comment:
can we make this a bit more explicit? Kinda gross to have this in the function call - here and elsewhere